src/Table.tsx

```diff
@@ -481,12 +481,29 @@ function Table<RecordType extends DefaultRecordType>(props: TableProps<RecordTyp
     }
   };
 
-  const onFullTableResize = ({ width }) => {
-    if (width !== componentWidth) {
-      triggerOnScroll();
-      setComponentWidth(fullTableRef.current ? fullTableRef.current.offsetWidth : width);
+  const requestAnimationFrameIdRef = React.useRef<number>();
+  const cancelRequestAnimationFrame = () => {
+    if (requestAnimationFrameIdRef.current) {
+      window.cancelAnimationFrame(requestAnimationFrameIdRef.current);
+      requestAnimationFrameIdRef.current = null;
     }
   };
+  React.useEffect(() => {
+    return () => {
+      cancelRequestAnimationFrame();
+    };
+  }, []);
+
+  const onFullTableResize = ({ width }) => {
+    // fix chrome throw ResizeObserver loop limit exceeded
+    cancelRequestAnimationFrame();
+    requestAnimationFrameIdRef.current = window.requestAnimationFrame(() => {
```

Comment:
It's by design to force render in sync to avoid table blink in next frame. You can wrap in Promise or `useLayoutEffect` instead.